commit da1644313d38f02fc90e2bb0fc01a4511cdbda29
Merge: 903b8bb ee60225
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Fri Sep 23 16:39:37 2016 -0500

    Merge pull request #4883 from esenapaj/Follow-up-the-PR-#4860
    
    Follow-up the PR #4860 (Fix BLTOUCH string and SERVO_DELAY)

diff --git a/Marlin/language_en.h b/Marlin/language_en.h
index c5ee084..ca2fd61 100644
--- a/Marlin/language_en.h
+++ b/Marlin/language_en.h
@@ -366,7 +366,7 @@
 #ifndef MSG_ZPROBE_OUT
   #define MSG_ZPROBE_OUT                      "Z probe out. bed"
 #endif
-#ifndef MSG_BLTOUCH_RESET
+#ifndef MSG_BLTOUCH_SELFTEST
   #define MSG_BLTOUCH_SELFTEST                "BLTouch Self-Test"
 #endif
 #ifndef MSG_BLTOUCH_RESET
